app/serializers/api/tracks_serializer.rb

```diff
@@ -0,0 +1,98 @@
+class API::TracksSerializer
+  include Rails.application.routes.url_helpers
+  include Mandate
+
+  def initialize(tracks, user = nil)
```

Comment:
Is the reason this `initialize` method has to be defined the fact that `user` can be `nil`? I thought `Mandate` generated this code

Reply:
Mandate doesn't have support (yet) for `nil` constructor params, and I don't want to have to pass `nil` into it in the places where it's not used. PRs to Mandate welcome to fix this 😉

Reply:
Hah!